src/linear_algebra/basic.lean

```diff
@@ -816,6 +816,8 @@ mem_sup.trans $ by simp only [set_like.exists, coe_mk]
 
 end
 
+/- This is the character `∙`, with escape sequence `\.`, and is thus different from the scalar
```

Comment:
This bit me this morning and was briefly very confusing. There is already a similar comment at the top of the file but I did not see it so I think it's worth having one here too.